tests/Eccube/Tests/Service/SystemServiceTest.php

```diff
@@ -27,11 +27,19 @@
 
 class SystemServiceTest extends AbstractServiceTestCase
 {
-    private $system;
-
     public function testgetDbversion()
     {
-        $system = $this->app['eccube.service.system'];
-        $this->assertNotNull($system->getDbversion());
+        $app = $this->app;
+
+        $version = $app['eccube.service.system']
+            ->getDbversion();
+
+        $this->assertNotNull($version);
+
+        if ($app['config']['database']['driver'] == 'pdo_mysql') {
+            $this->assertRegExp('/mysql/', strtolower($version));
+        } else {
```

Comment:
__[phpmd]__
* The method testgetDbversion uses an else expression. Else is never necessary and you can simplify the code to work without else.